src/components/desktop/desktop.vue

```diff
@@ -37,7 +37,9 @@
       </div>
     </template>
 
-    <template #sub-header>
+    <template #sub-header v-if="!$x.redirections.length">
+      <SpellcheckMessage class="x-margin--bottom-05" data-test="spellcheck-message" />
+      <NoResultsMessage class="x-margin--bottom-05" data-test="no-results-message" />
```

Comment:
What would you think of wrapping this inside a `LocationProvider` to keep sending the `results` location? (and the same for mobile).

I'm not sure either if it is really relevant in this case, but I think I would add it so the archetype is more complete.

Reply:
I would say that the correct Location would be `no_results`  as described in the types:

https://github.com/empathyco/x/blob/9c235bc9457cc959fa2a386b8dce114ff3bf6df7/packages/x-components/src/types/origin.ts#L62

And correct origin would be `spellcheck`:

https://github.com/empathyco/x/blob/9c235bc9457cc959fa2a386b8dce114ff3bf6df7/packages/x-components/src/types/origin.ts#L32